### ISSUE TITLE
Return key no longer sends message on iOS/iPad app

### ISSUE THREAD

**Issue by alextnewman** [keybase/client#25143]
Previously, the Return key on either a physical iPad Keyboard or Apple Silicon MacBook would send a message in the iPad client. As of a recent update, the only way to send a message is to manually click the Send button in the UI. Can the Return key functionality be restored?

**Comment by gjiva** [comment]
Yeah, this is super annoying, especially on macOS. At the very least, it should be a preference toggle.
